airmozilla/main/static/main/js/calendar.js

```diff
@@ -1,31 +1,44 @@
 $(function() {
-    var weekends, firstDay = 0;
-    var x = new Date();
-    var currentTimeZoneOffsetInHours = x.getTimezoneOffset() / 60;
-    if (currentTimeZoneOffsetInHours <= -7) {
-        // sufficiently far east that some Friday events might actually
-        // start on a Saturday
-        weekends = true;
-        firstDay = 1;  // far-east people prefer Monday
-    } else {
-        weekends = false;
-    }
+    var weekends = true;
     var container = $('#calendar');
-    container.fullCalendar({
-        events: {
-            url: container.data('events-url'),
-            cache: true
-        },
-        timezone: 'local',
-        weekends: weekends,
-        firstDay: firstDay,
-        allDaySlot: false,
-        defaultTimedEventDuration: '01:00:00',
-        timeFormat: 'h:mm a',
-        header: {
-            left: 'prev,next today',
-            center: 'title',
-            right: 'month,agendaWeek,agendaDay'
+    var startsOnMonday = $('#startsOnMonday');
+    var storage_startsOnMonday = localStorage.getItem('startsOnMonday');
+    if (storage_startsOnMonday === null) {
+        // this is a hack to try to determine if week starts on Sunday or Monday
+        // right now if user is in America's TZ the week would start on Sunday
+        // if user is in another TZ It would start on Monday
+        var tz = jstz.determine().name();
+        if (!tz.match('^America/')) {
+            startsOnMonday.prop('checked', true);
         }
+    } else {
+        if (storage_startsOnMonday == 1) {
```

Comment:
This is an ugly side of javascript. It's actually doing `if ("1" == 1) {` and it becomes `true`. 
It works to our advantage in this case. :)